src/Logging/Logging.Console/Output/UnicodeColor.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace CodeMonkeys.Logging.Console
+{
+    internal readonly struct UnicodeColor
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Implement 'IEquatable<T>' in value type 'UnicodeColor'.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)